mixins/_typography.pcss

```diff
@@ -286,14 +286,26 @@
  * ----------------------------------------------------------------------------- */
 
 @define-mixin anchor-thin-alt {
-	border-bottom: 1px solid var(--color-accent-primary);
-	color: var(--color-text-primary);
-	transition: var(--transition-color);
+	border-bottom: 1px solid var(--tec-color-link-accent);
+	color: var(--tec-color-link-primary);
+	transition: var(--tec-transition-color);
 
 	&:active,
 	&:focus,
 	&:hover {
-		border-bottom: 1px solid var(--color-accent-primary);
-		color: var(--color-accent-primary);
+		border-bottom: 1px solid currentColor;
+		color: var(--tec-color-link-accent);
+	}
+}
+
+@define-mixin event-title-link {
+	color: var(--tec-color-text-events-title);
+
+	&:active,
+	&:focus,
+	&:hover,
+	&:visited {
+		border-color: currentColor;
+		color: var(--tec-color-text-events-title);
 	}
```

Comment:
Split the styles and added header.

@paulmskim 
As for #1 - do you mean a comment/docblock? Since the underline of the event title link depends on where it is displayed, we're just handling the color here.

Reply:
no, I mean shouldn't it look something like:

```
@define-mixin event-title-link {
	border-bottom: 2px solid transparent;
	color: var(--tec-color-text-events-title);

	&:active,
	&:focus,
	&:hover {
		border-color: currentColor;
		color: var(--tec-color-text-events-title);
	}
}
```

This way, the border is part of the mixin rather than having to define it separately. It feels weird if the dependency needed to be defined separately from the mixin itself.

Reply:
I like the idea - but this is meant to just be a color override for event titles specifically. (maybe it needs a better "docblock"?)

The border width is handled in a separate mixin - `anchor-thin` for example, which specifies a color but assume the link is `--color-text-primary` and the color will match.

The problem with that was the Customizer would change the text color but not the border color. The border color should always match, so using currentColor seemed more appropriate.

🤔 Should we instead modify the anchor mixins?

Reply:
Note - this would apply to both (real-world examples) `tribe-events-calendar-month-mobile-events__mobile-event-title-link **tribe-common-anchor**` (2px width) and `tribe-events-calendar-month__calendar-event-title-link **tribe-common-anchor-thin**` (1px width)